Bottom: ae6374f4f52b047b3f17707bae498916bba862d7
Top:    a0c673156cb90c6eb58a8b7bee0cc85c00411a74
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-02-02 14:49:28 +0300

Refresh of ctl-verbs.patch

---

diff --git a/srv.c b/srv.c
index d932969..ecb8ad8 100644
--- a/srv.c
+++ b/srv.c
@@ -7,7 +7,7 @@ srv_init(void)
 {
 	struct IB_SOCK *sock, *sock_child = NULL;
 	unsigned long event = 0;
-	bool read_st;
+	bool read_st = false;
 	void *data;
 	int ret;
